config/app.php

```diff
@@ -244,14 +243,12 @@
     | this array to grant expanded functionality to your applications.
     |
     */
-
-    'providers' => array_merge(include(base_path('modules/system/providers.php')), [
-
+    
+    'providers' => array_merge(include base_path('modules/system/providers.php'), [
```

Comment:
@jaxwilko looks like the printer / parser barfs on nested functions, we'll need to support that

Reply:
Added with backtracing to ensure includes were () correctly dependent upon position: https://github.com/wintercms/storm/pull/40/commits/c098853f8e082e4a29e73c8dc21d873195337777
Removed above and simply always appended (): https://github.com/wintercms/storm/pull/40/commits/892ed2a1dd87e4bb75adce332fe96288fcf16260